Assets/Scripts/MarchingCubes/Chunk.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public class Chunk : MonoBehaviour
+{
+
+    int index;
+    int resolution;
+    MeshFilter meshFilter;
+    MeshCollider meshCollider;
+    Mesh mesh;
+    MarchingCubes marchingCubes;
+    PillPlayerController player;
```

Comment:
Not used but creates a potential future dependency on PillPlayerController.

Reply:
Will be used in Chunks later on to do "LOD's"

Reply:
Change it to GameObject or even Transform of player to keep it abstract?

Reply:
Goes for all similar comments

Reply:
fixed